### DISCUSSION TITLE
my spacebar does not work when i open a notebook

### DISCUSSION THREAD

**Discussion by neo451** [microsoft/vscode-jupyter#14279] [Questions and Answers] [answered]
the markdown notebook has the same issue, everything else works but i just can not make any space using spacebar.

**Comment by chrisbolin** [comment, marked as answer]
I've seen this issue, too. For me it was caused by another extension that I had had for months before the Jupyter extension. The bug did not surface until installing Jupyter. 
1. Open Keyboard Shortcuts in VSCode
2. Search for `"space"` (with the double-quotes)
3. Try disabling any whose **Source** is not `System`

This is caused by a poorly defined **When**. Once you find the offending item you can edit the **When**.

**Reply by neo451** [reply]
thank you for your reply, I forgot to close this, your solution is correct indeed

**Reply by chrisbolin** [reply]
@neo451 the offending extension for me was [TidalCycles for VSCode](https://marketplace.visualstudio.com/items?itemName=tidalcycles.vscode-tidalcycles). Maybe it was the same for you? I noticed you make music :)

**Reply by neo451** [reply]
yes it is! you are :100: right lol.

**Reply by kaosbeat** [reply]
also came here from tidalcycles + jupyter!
Should this be reported in the tidalcycles ext or is it a jupyter bug?
